src/anonymous-credentials/schema.ts

```diff
@@ -1054,6 +1071,8 @@ export class CredentialSchema extends Versioned {
         obj.properties[k] = CredentialSchema.getSubschema(v);
       }
       return obj;
+    } else if (typ === 'boolean') {
```

Comment:
Better to move it to above check of `typ === 'string'` to reduce if-else branches

Reply:
this whole method can benefit from removing the "else" to make it a bit clearer as it always returns. updated for that